pb-crm-ui/src/app/invoicing/services/account.service.ts

```diff
@@ -0,0 +1,52 @@
+import {Injectable} from '@angular/core';
+import {Observable} from 'rxjs';
+import {ApiService} from '../../api/api.service';
+import {GetAllAccountResponse} from '@hiddentemple/api-interfaces';
+import {GetAccountResponse} from '@hiddentemple/api-interfaces';
+import {
+  AccountModel,
+  CreateAccountRequest,
+  CreateAccountResponse, UpdateAccountRequest, UpdateAccountResponse
+} from '@hiddentemple/api-interfaces';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ContactService {
```

Comment:
All the routes in this class are pointing to contact. Also this class is not named correctly.